src/view/drag-handle/util/create-scheduler.js

```diff
@@ -1,44 +1,32 @@
 // @flow
 import memoizeOne from 'memoize-one';
-import rafSchedule from 'raf-schd';
+import rafSchd from 'raf-schd';
 import type { Position } from '../../../types';
 import type { Callbacks } from '../drag-handle-types';
 
-export default (callbacks: Callbacks, isDraggingFn: () => boolean) => {
-  const ifDragging = (fn: Function) => {
-    if (isDraggingFn()) {
-      fn();
-    }
-  };
-
+export default (callbacks: Callbacks) => {
   const memoizedMove = memoizeOne((x: number, y: number) => {
     const point: Position = { x, y };
     callbacks.onMove(point);
   });
 
-  const move = rafSchedule((point: Position) => {
-    ifDragging(() => memoizedMove(point.x, point.y));
```

Comment:
Rather than executing these checks every time, no we just cancel pending frames when they are no longer needed